### PR TITLE
fix(desktop): 修复 CJK 全角标点相邻时加粗星号解析失败

### PR DESCRIPTION
**Pull request by jinguochen-dev**
### 摘要

桌面端聊天消息里,AI 输出的加粗标记 `**` 在紧贴中文全角标点时无法解析,正文残留原始星号(如 `**注意：**`、`**“术语”**`、`**（注）**`)。原因是 CommonMark 的 emphasis 侧翼(flanking)规则把全角标点算作「标点」:加粗内容以标点结尾、右侧 `**` 后又紧跟普通字符时,闭判定不成立(开判定对称),整对星号退化成字面量。本 PR 在 MarkdownRenderer 的两条 remark 插件链中、于 remarkGfm 之后注册 `remark-cjk-friendly`(micromark 语法扩展,放宽该判定:全角标点不再当标点),修复全部此类写法,并与 mobile 自研 parser 的既有行为对齐(端间差异消除)。

### 变更类型

- [x] `fix` 缺陷修复

### 范围

- 关联 Issue / 需求：无(用户反馈:流式输出正文残留原始 `**`)
- 本 PR 包含：desktop 新增依赖 `remark-cjk-friendly@^2.3.1`;MarkdownRenderer 普通/受信任两条插件链注册该插件;新增 19 个单元测试
- 明确不包含：`~~` 删除线的同类全角标点问题(gfm strikethrough 有独立定界逻辑,本 PR 前后行为一致,保持现状);mobile 无改动(其 parser 本就能解析这些写法)
- 用户可见变化：中文回复里紧贴全角标点的加粗现在正确渲染为加粗,不再显示原始 `**` 字符
- 是否存在 breaking change：无

### UI 变化

before:正文残留字面 `**注意：**`;after:渲染为加粗「注意：」。仅修复解析,无新增视觉样式、交互或文案变化。未附截图——本次在 worktree 会话内开发,改动不进入运行中的 dev 实例,无法实机截图;解析结果已由管线级渲染测试覆盖(见下)。

- 引用的设计规范：docs/design-rules/DESIGN.md 的 Markdown 文本色条款(`--md-strong-fg`,strong 的既有渲染规范);本 PR 只是让命中文本恢复进入该既有样式

### 自动验证

```text
corepack pnpm exec vitest run src/renderer/__tests__/markdownCjkStrongDelimiter.test.ts src/renderer/__tests__/markdownStrikethrough.test.ts
结果：2 个文件 23 个测试全部通过(新增 19 个:8 个历史失败样例 → <strong>;其余为守卫:带空格/转义/行内代码/链接地址内的星号保持字面量,~~ 删除线与公式行为不变,source-contract 锚定两条插件链的注册)

corepack pnpm --filter desktop run typecheck
结果：通过,无错误

corepack pnpm test:unit(仓库根全量)
结果：rebase 到含 #1034 的最新主干后复跑——runner 自测全过;mobile 及全部 packages workspace 通过;desktop 仅 scriptRunnerPythonProtocol 3 个失败(本机 Python 环境问题,干净 upstream/main 上复跑同样失败,与本 PR 无关;#1034 已修复此前其余 Windows 本机环境用例);本次新增的 19 个测试在全量中全部通过

补充:首次全量跑于旧主干时另有 13 个本机环境失败(newMakerProjectPicker、endpointManifestCache、money 等),已在干净主干验证为既有问题,且已被上游 #1034 修复

corepack pnpm install --frozen-lockfile
结果：通过,lockfile 与 package.json 一致
```

### 手工验证

不涉及(worktree 会话内改动不进入运行中的 dev 实例,无法实机目检;建议合并前在 dev 实例发送含 `**注意：**` 的消息复核)

### 未执行的验证

- 实机渲染目检:原因如上,已由管线级 renderToStaticMarkup 测试覆盖解析结果
- mobile 验证:mobile 无改动

### 风险分类

- [x] 无已知风险

### 影响与回滚

- 影响范围：desktop 所有走 MarkdownRenderer 的 Markdown 渲染(聊天气泡、计划卡片等)。remark-cjk-friendly 仅放宽 emphasis/strong 的定界判定,不碰 `~~` 删除线、公式、代码、链接(均有测试守卫);不改动源文本,doc 模式行号锚点不受影响
- 回滚 / 降级方式：revert 本 PR 即可恢复原行为

### 提交前检查

- [x] 已 review 完整 diff
- [x] 每个 commit 都带 DCO 签名(`git commit -s`,见 [DCO](../DCO))
- [x] UI 改动已在「UI 变化」注明引用的设计规范章节(不涉及 UI 则跳过)
- [x] 未提交凭证、令牌或授权文件
- [x] 已补充必要文档(代码注释与测试文件头说明,无需额外文档)
- [x] 已确认测试结果或说明未执行原因

### CODE REVIEW

**Comment by MagicLizi** [comment]
@jinguochen-dev 👋 这个 PR 还有 1 条 review conversation 没 resolve(apps/desktop/package.json),auto-review 因此暂时跳过、没法继续审查 / 合并。

如果你已经按评论改完或回应了,请到对应 thread 上点 **Resolve conversation**;全部 resolve 后,下一轮 auto-review 会自动重新审查这个 PR。

File: apps/desktop/package.json [hunk 1]
```diff
@@ -153,6 +153,7 @@
     "rehype-highlight": "^7.0.2",
     "rehype-katex": "^7.0.1",
     "rehype-slug": "^6.0.0",
+    "remark-cjk-friendly": "^2.3.1",
```
[the remark this answers is not in the document]
Comment (jinguochen-dev) [reply apps/desktop/package.json:156]: 已处理:运行了 `pnpm licenses:generate` 并提交全部生成物(b4928d54)。npm 包数 773 → 778,新增的 5 个均为 remark-cjk-friendly 及其传递依赖(get-east-asian-width、micromark-extension-cjk-friendly(-util)、mdast-util-to-markdown-cjk-friendly)的 MIT 声明;docs/legal/notices 与 apps/desktop/resources 下的声明与 SBOM 已同步,mobile SBOM 仅生成时间戳变化。

**Comment by MagicLizi** [comment]
⏸️ 本 PR 新增第三方依赖 `remark-cjk-friendly`,触发 security 签字门,已开讨论 issue 等待 roster 确认。如需放行:在 PR 评论或讨论 issue 里回复同意,或摘掉 `need-whitelist` 标签。

讨论 issue:<https://github.com/makecindy/cindy/issues/1085>

**Comment by MagicLizi** [comment]
本 PR 命中 UI 路径但 description 未附界面效果证据——建议补充改动后效果：截图/录屏，或改动后界面的 HTML 页面（代码块、.html 附件或在线预览链接），便于确认界面符合 DESIGN.md 设计规范。

此提醒不阻断审查和合并，只是建议补充。

File: apps/desktop/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MagicLizi) [reply apps/desktop/package.json:156]: 已确认:commit b4928d5 在评论后提交,已运行 `pnpm licenses:generate` 并收录 remark-cjk-friendly 及其传递依赖。第三方声明已重新生成。

**Review by MagicLizi — APPROVED** [review]
在「待确认 PR」面板确认放行。

**Comment by MagicLizi** [comment]
白名单确认门已放行(product / security / arch,由 MagicLizi 确认),PR 恢复正常推进。

**Review by MagicLizi — APPROVED** [review]
Review passed (heavy tier) — clean integration of remark-cjk-friendly plugin. All 20 security soft hits confirmed as false positives (no ReDoS risk, no XSS vector, trusted authors). Comprehensive test coverage for edge cases.

⚠️ PR has merge conflicts — please rebase or merge main before this can land.

**Comment by MagicLizi** [comment]
@jinguochen-dev 👋 这个 PR 目前与 `main` 有合并冲突,auto-review 因此暂时跳过、没法继续审查 / 合并。

请在本地 merge 最新的 `origin/main` 解决冲突后推送;冲突解除后,下一轮 auto-review 会自动重新处理这个 PR。

**Review by MagicLizi — APPROVED** [review]
Auto-review: heavy-tier 供应链安全审查通过。14 文件 / 2401 行。
- 新依赖 remark-cjk-friendly v2.3.1:MIT,tats-u 维护,8M+ 月下载,GitHub Actions OIDC provenance
- 765 soft hits 来自 THIRD-PARTY-NOTICES 自动生成文本(license 样板),非安全问题
- 依赖纯 AST 操作,无 I/O / 无 eval / 无网络
- 测试覆盖:8 回归 + 4 非回归 + 5 边界

零 P0/P1。注意:PR 可能与 main 有 THIRD-PARTY-NOTICES 冲突,需作者解决后重试合并。

**Review by MagicLizi — APPROVED** [review]
✅ **Auto-review (heavy tier)**: PASS

审查完成,零问题。

- 安全:`remark-cjk-friendly@2.3.1` 是纯算法 micromark 扩展(MIT,155 stars),无网络/文件系统操作,无 install scripts,依赖链全 MIT
- 恶意代码检查:通过
- 正确性:修复准确——CJK 全角标点被 CommonMark emphasis flanking 规则误判为标点,插件正确排除
- 架构:最小改动,两处插件数组插入,位置正确(remarkGfm 之后)
- 测试:19 条覆盖历史失败用例 + 回归防护 + 源码合约测试

⚠️ 当前与 main 有合并冲突(MarkdownRenderer.tsx、package.json 等),请 rebase 后下一轮即可合并。

**Comment by MagicLizi** [comment]
解决了一个用户实际痛点 — CJK 用户平时写加粗文字总踩到这个坑。remark-cjk-friendly 选型恰当，测试覆盖也很全面，包括负面用例。主干侧代合了锁文件冲突，合进来了。
